ironfish-cli/src/commands/peers/index.ts

```diff
@@ -95,13 +88,11 @@ export class ListCommand extends IronfishCommand {
 function renderTable(
   content: GetPeersResponse,
   flags: {
-    extended: boolean
     names: boolean
     all: boolean
-    sort: string
     agents: boolean
     sequence: boolean
-  },
+  } & ReturnType<typeof CliUx.ux.table.flags>,
```

Comment:
It might make sense to change the `flags` that are currently passed in to `options` and have a separate `flags` argument for the table flags

I think the lint errors are coming from passing these booleans into `Cli.ux.table`

Reply:
Fixed this by adding a new CommandFlags type.